projects/observability/src/shared/components/bar-gauge/bar-gauge.component.ts

```diff
@@ -26,9 +34,9 @@ import { DomElementMeasurerService, getPercentage, TypedSimpleChanges } from '@h
           <div class="segment-bars">
             <div
               #segmentBars
-              *ngFor="let segment of this.barSegments"
+              *ngFor="let segment of this.barSegments; let index = index"
```

Comment:
Doesn't look like index is used (but it reminds me, you could use `last` like `let last = last` and then change the condition to something like `last && this.isGaugeFull`- I took liberties with the name, could also leave it)

Reply:
Ah, thanks. That was left over from one of my other attempts. I'll remove.